### PR TITLE
Style models update

### PR DESCRIPTION
**Pull request by shixiedesign**
### Related Ticket(s)

#1384

### Description

- Simplification of style models from 9 to 3
- Reduced content from 2 pages (2 tabs) to just 1 page no tab

### Changelog

**New**
- `Guidelines / style-models`

**Changed**
- `Data / nav-items.yaml`

**Removed**

- `Guidelines / Style models / overview`
- `Guidelines / Style models / style-model-list`
- And associated images

### CODE REVIEW

**Review by janchild — COMMENTED** [review]
Hey @shixiedesign, this is great! Such an improvement. I have a few little things but nothing major. I really like the flow of this page. It's great! Let's wrap it up and ship it! 🚀

File: src/pages/guidelines/style-models.mdx [hunk 1]
```diff
@@ -0,0 +1,309 @@
...
+</Column>
+</Row>
+
+### Screen regions and Dotcom shell
+
+Reference the 2x Grid <a href="https://carbondesignsystem.com/guidelines/2x-grid/overview/#screen-regions" target="_blank">screen regions</a> guidance on Carbon website for details.
+
+This is style model supports high information density moments, and at IBM.com, we have not yet encountered this use case. If you content fits this cateogry, and is not external facing, consider using simply using Carbon Design System, where UI shell is the default header in place of Dotcom shell.
+
+If Dotcom shell is used for some reason, follows the grid and stretches to full width.
```
[the remark this answers is not in the document]
Comment (shixiedesign) [reply src/pages/guidelines/style-models.mdx]: Yep the content was a repeat. I’ve updated it now!

**Comment by shixiedesign** [comment]
Ready for another look @janchild

**Review by janchild — APPROVED** [review]
🥂 🚀

**Pull request merged**